config/datasetIngest.py

```diff
@@ -1,5 +1,3 @@
 config.dataFiles = ['*.fits.gz']
 
-config.defectTarball = 'defects.tar.gz'
-
```

Comment:
IMO this should have been part of the ticket that changed `DatasetIngestConfig`, but I guess it's too late for that now.